### ISSUE TITLE
robe-start fails to load pry

### ISSUE THREAD

**Issue by gozes** [dgutov/robe#50]
after installing robe and doing bundle init, added pry and pry-doc to dev dependency and bundle install I get
 LoadError: cannot load such file -- pry
	from /home/gozes/.emacs.d/elpa/robe-20150126.808/lib/robe/sash/doc_for.rb:1:in `require'
	from /home/gozes/.emacs.d/elpa/robe-20150126.808/lib/robe/sash/doc_for.rb:1:in `<top (required)>'
	from /home/gozes/.emacs.d/elpa/robe-20150126.808/lib/robe/sash.rb:1:in `require'
	from /home/gozes/.emacs.d/elpa/robe-20150126.808/lib/robe/sash.rb:1:in `<top (required)>'
	from /home/gozes/.emacs.d/elpa/robe-20150126.808/lib/robe.rb:1:in `require'
	from /home/gozes/.emacs.d/elpa/robe-20150126.808/lib/robe.rb:1:in `<top (required)>'

OS: Linux
Ruby Version: 2.1
RVM: NO

**Comment by dgutov** [comment]
Dev dependency where?

**Comment by gozes** [comment]
pry, pry-doc, bundler and rake.

**Comment by dgutov** [comment]
Where do you specify them?

**Comment by gozes** [comment]
gemspec

**Comment by dgutov** [comment]
Do you have a `Gemfile`? What does it contain?

**Comment by gozes** [comment]
it just points to the gemspec. again the only thing i did after bundle init was add pry and pry-doc to my gemspec run bundle install and then run robo-start in emacs. there is were all went :boom:

**Comment by dgutov** [comment]
Work for me. Please post a project when you can see this error.

**Comment by Nollo** [comment]
Didyou try to do a global installation of pry "gem install pry" instead of putting it into your gemfile ?

**Comment by dgutov** [comment]
@Nollo If the console is launched using Bundler (which it should), this won't help.

**Comment by natetech333** [comment]
I learned that pry crashes with puma server. It freezes after debugging (next, continue). I am not assuming this is a problem with this plugin calling pry. Because I doubt this plugin is using the next and continue operations on pry. But it needs to be known in case you do run into weird errors. I see that the plugin runs pry in the background and talks to it. I had to switch to byebug with puma for debugging.
